### DISCUSSION TITLE
Add Control+F to activate search

### DISCUSSION THREAD

**Discussion by eugenesvk** [ente-io/ente#1254] [Enhancements]
Desktop Ente Auth app on Windows

Though this should also allow the user to add a different shortcut in addition (or instead of) this default

**Comment by glitch182** [comment]
+1 to this on the Mac version as well with a CMD+F shortcut.

**Comment by lunyx** [comment]
Does there even need to be a keyboard shortcut?  I don't see much reason to be typing with the ente auth window active unless you're trying to search.  Perhaps it should just start searching automatically when you start typing.  I had mentioned this suggestion here https://github.com/ente-io/ente/issues/2262#issuecomment-2310182787

**Comment by AmanRajSinghMourya** [comment]
This feature will be available in the next update. [#4552 ](https://github.com/ente-io/ente/pull/4552)

**Reply by JoelAzaria** [reply]
Hey now!   Came here to request this.   Very happy to see the rate of development and adoption.   

Authy who?   just a dot in my rearview mirror....

Big ups to the team here and keep up the good work!

**Reply by JoelAzaria** [reply]
Having the shortcut is a great help but it would be more helpful if when using the shortcut, any text in the 'find' field would be auto-highlighted.   

That way users don't have to manually highlight/delete the text to type a new name.

**Reply by lucasstinis** [reply]
UX-er here: when the search bar is selected (using a keyboard shortcut), **the current content should automatically be selected** so the user can easily type over it instead of first having to empty the field first...

This is might seem like "no big deal", but it's one of those "details" that _severely_ affects the usability of any app. In the case of Ente, this behaviour would arguably be preferable in 99,9% of use cases!

# Current behaviour
1. User has previously typed something in the search field.
2. A) User presses the key combination for search -or- B) selects the search field with the mouse

### Result:
- A) The cursor appears at the end of the current field content -or- B) the cursor appears where the user clicks
- The content of the field remains the same and nothing is selected, unless the user A) uses a key combination to "select all" -or- B) double clicks

# Desired behaviour
1. User has previously typed something in the search field.
2. A) User presses the key combination for search -or- B) selects the search field with the mouse

### Result:
- The entire contents of the field is selected and the cursor is positioned at the end of said content.
- When the user types, the entire contents is overwritten.
